mapie/tests/test_common.py

```diff
@@ -158,7 +158,7 @@ def test_invalid_method(MapieEstimator: BaseEstimator, method: str) -> None:
 
 @pytest.mark.parametrize("MapieEstimator", MapieSimpleEstimators())
 @pytest.mark.parametrize(
-    "cv", [-3.14, -2, 0, 1, "cv", LinearRegression(), [1, 2]]
+    "cv", [-3.14, -2, 1, 0, "cv", LinearRegression(), [2, 3]]
```

Comment:
why this change ?

Reply:
Oh! I think it is a modification I made when testing my changes. It is no pertinent indeed.

Reply:
Ok, let's come back to the original

Reply:
Done.